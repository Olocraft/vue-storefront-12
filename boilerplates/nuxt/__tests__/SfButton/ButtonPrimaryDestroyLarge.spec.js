import { shallowMount } from '@vue/test-utils'
import ButtonDestroyLarge from '../../components/SfButton/Primary/Destroy/ButtonDestroyLarge.vue'

let wrapper

describe('ButtonDestroyLarge.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(ButtonDestroyLarge)
  })
  it('render component correctly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
