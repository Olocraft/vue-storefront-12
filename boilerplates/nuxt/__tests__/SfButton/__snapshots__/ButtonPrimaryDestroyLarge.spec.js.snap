// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonDestroyLarge.vue render component correctly 1`] = `<button class="inline-flex items-center justify-center px-6 py-3 text-base font-medium text-white uppercase rounded-md shadow cursor-pointer bg-rose-600 hover:bg-rose-700 active:bg-rose-800 disabled:bg-gray-200 disabled:text-gray-500 disabled:opacity-50 font-body hover:shadow-md disabled:shadow-none disabled:cursor-not-allowed outline-violet"></button>`;
